streams/src/main/java/org/apache/kafka/streams/kstream/KGroupedTable.java

```diff
@@ -74,19 +74,19 @@
      * will be automatically created in Kafka for failure recovery, where "applicationID"
      * is specified by the user in {@link org.apache.kafka.streams.StreamsConfig}.
      *
+     * @param <T>           the value type of the aggregated {@link KTable}
```

Comment:
Any reason why you have moved these type param? They are consistently the last params in all of our javadoc.